src/main/java/movies/Main.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * @(#)Main.java
+ *
+ * Copyright (c) 2018 Jala Foundation.
+ * 2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+/**
+ * Main class for testing
+ * @version 1.0.
+ */
+package movies;
+
+import movies.movie.Movie;
+import movies.movie.NewReleaseMovie;
+import movies.movie.RegularMovie;
+
+public class Main {
+    public static void main(String[] args) {
+        Customer customer = new Customer("Test");
+        customer.addRental(new Rental(new NewReleaseMovie("The Revenant"), 2));
+        customer.addRental(new Rental(new RegularMovie("Terminator"), 2));
+        System.out.println(customer.statement());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)